ftw/news/browser/templates/news_listing_block.pt

```diff
@@ -34,6 +34,11 @@
                                           i18n:name="author">Author</span>
                                 </span>
                             </tal:author>
+                            <tal:review_state condition="item/review_state/show_review_state">
+                                <span>| <span tal:content="item/review_state/review_state_title"
```

Comment:
Can we move the pip symbol into a css pseudo element? Depending on the Theme there will be another symbol and we don't have to override the whole template in order to change the symbol.